app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -1003,6 +1003,16 @@ def build_toolbar_disable_button(id)
       when "container_group_timeline"
         return "No Timeline data has been collected for this Pod" unless @record.has_events? || @record.has_events?(:policy_events)
       end
+    when "ManageIQ::Providers::Kubernetes::ContainerManager::ContainerProject"
+      case id
+      when "container_project_timeline"
```

Comment:
> @dkron you can add tests for button availability of the newly added buttons here, look at https://github.com/ManageIQ/manageiq/blob/master/spec/helpers/application_helper/toolbar_builder_spec.rb#L181 for examples.

I'm not sure I understand what addition is needed there, can you please clarify or give me another example?